tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/AbstractTychoMapping.java

```diff
@@ -52,8 +52,12 @@
  */
 public abstract class AbstractTychoMapping implements Mapping, ModelReader {
 
-    private static final String TYCHO_POMLESS_PARENT = "tycho.pomless.parent";
-    private static final String PARENT_POM_DEFAULT_VALUE = System.getProperty(TYCHO_POMLESS_PARENT, "..");
+    // All build.properties entries specifically considered by Tycho. Extends the list in Mapping interface
+    protected static final String TYCHO_POMLESS_PARENT_PROPERTY = "tycho.pomless.parent";
+    protected static final String TYCHO_POMLESS_AGGREGATOR_NAMES_PROPERTY = "tycho.pomless.aggregator.names";
+    protected static final String TYCHO_POMLESS_TESTBUNDLE_PROPERTY = "tycho.pomless.testbundle";
```

Comment:
Please remove the property and document it in the migration guide.

Reply:
Removed it.
Documentation in the Release-notes? Or is there a dedicated guide? is still pending.
Have to mention that `pom.model.packaging = eclipse-test-plugin` can be used already.